<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Face</class>
 <widget class="QWidget" name="Face">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Mine_btn">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>我的信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="history_btn">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>历史订单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UnCheck_btn">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>未验收订单</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
